### ISSUE TITLE
Uber emails need an "Unsubscribe" button

### ISSUE THREAD

**Issue by kitsuta** [magfest/ubersystem#993]
In the past, because we send emails directly through uber and only to registered attendees, we've considered ourselves good as far as spamming compliance goes.

However:
1. A handful of attendees this year hit the "Spam" button to report our confirmation emails for unknown reasons.
2. We definitely had a case of someone who registered using the wrong email address, and the person who got the emails was not... super... happy about it.

Basically, we're getting to the point where we should let people tell us they don't want our emails anymore - even though this should technically never happen.

I'm thinking we should have an email footer that says something like, "These emails are being sent to you because you are registered for MAGFest 2016. If you're sure that this doesn't apply to you, please [unsubscribe] to opt out of all future emails (and your badge)."

The unsubscribe link would:
1. Reset the attendee's email address in the system.
2. Notify the registration admin email that the attendee unsubscribed.

That way we can take further action (invalidate the badge, notify the group leader if there is one, etc).

**Comment by thaeli** [comment]
Agreed. We should probably send our emails through an ESP and API at this point anyway.
